src/DependenciesPackager/ProcessRunner.cs

```diff
@@ -96,9 +96,11 @@ public int Run()
             }
 
             var processInfo = CreateProcessInfo();
-            _process = new Process();
-            _process.StartInfo = processInfo;
-            _process.EnableRaisingEvents = true;
+            _process = new Process
+                       {
```

Comment:
Just indent by 4 spaces? I actually don't care that much where the `{` is, but it's weird that the property setters within it are not indented 4 spaces more than wherever the `{` is.